src/Grid/Col/index.js

```diff
@@ -12,12 +15,18 @@ export default function Col({ children, col, offset, style }) {
 Col.propTypes = {
   children: PropTypes.node.isRequired,
   style: PropTypes.shape({}),
-  col: PropTypes.number,
-  offset: PropTypes.number
+  offset: PropTypes.number,
+  xs: PropTypes.oneOf([0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, "none"]),
```

Comment:
is none apart of the spec for PropTypes.oneOf?

Reply:
none you can use when you wanna hide a column in one of the screens. So for example in you want in Tablet to hide a Column or something then you should use `sm="none"`

Reply:
Why not “hide”?

Reply:
```
  .col-lg-none{
    display: none;
  }
```

this is the class that would make the display none. I thought it is close naming to the native css classes so thats why. 
If it still doesnt make sense I dont mind renaming it